### ISSUE TITLE
Add more traits (weight etc)

### ISSUE THREAD

**Issue by auserwithaname** [il2planes/il2planes.github.io#1]
Hey, not sure if you're open to suggestions or issues, but are you open to adding weights?

**Comment by unites** [comment]
Yep, I can do that fairly easily.  I am actually in the process of moving this whole thing onto a web framework, I will try to add more info on ammo and weight when I do.
